lib/search_vector.rb

```diff
@@ -5,10 +5,13 @@ module SearchVector
   AcceptedTables = ['supporters', 'payments']
 
   def self.query(query_string, expr=nil)
-    (expr || Qexpr.new).where(
+    query = if query_string.is_a?(Integer) || query_string.is_int?
+      "payments.id = $search"
+    else
       "(supporters.fts @@ websearch_to_tsquery('english', $search)
-      OR donations.fts  @@ websearch_to_tsquery('english', $search))",
-      { search: query_string }
```

Comment:
When an integer is passing, I think we should still be searching in the `supporter.fts` or `donations.fts` in addition to the `payments.id`.